tst/EnergyPlus/unit/EMSManager.unit.cc

```diff
@@ -1558,5 +1571,39 @@ TEST_F(EnergyPlusFixture, EMSManager_TestOANodeAsActuators)
     EXPECT_TRUE(Node(1).IsLocalNode);
     EXPECT_FALSE(Node(2).IsLocalNode);
     EXPECT_TRUE(Node(3).IsLocalNode);
-
+}
+TEST_F(EnergyPlusFixture, EMSManager_TestWindowShadingControlExteriorScreenOption2b)
```

Comment:
This unit test is set up nicely.  I think it needs a change at the end, but it is succinct and clearly set up.  When you make the change to the test expectations at the bottom, please also rename the test (no need for the 2b anymore)

Reply:
@Myoldmopar I am going to make the change by removing "2b".